layouts/shortcodes/docs/languages/exporters-intro.md

```diff
@@ -1,22 +1,34 @@
-<!-- cSpell:ignore isset cond -->
+{{ $lang := .Get 0 -}}
 
-{{ $lang := .Get 0 -}} {{ $data := index $.Site.Data.instrumentation $lang }}
-{{ $name := cond (isset $.Site.Data.instrumentation $lang) (printf "OpenTelemetry %s" $data.name) "the language specific implementations" -}}
+{{ $data := index $.Site.Data.instrumentation $lang -}}
 
-In order to visualize and analyze your telemetry, you will need to export your
-data to an [OpenTelemetry Collector](/docs/collector/) or a backend such as
+{{ $name := $data.name -}}
+
+To visualize and analyze telemetry, export it to the
```

Comment:
if we want to be precise - then we should revisit this... This gives the impression that Collector is a  tool for "visualizing and analyzing", but it is not. Not introduced in this PR, so not a blocker for this.

Reply:
Let's track this via:

- #3951

Thanks for the review and approval @cijothomas.